### ISSUE TITLE
Fatal error while importing variant select field

### ISSUE THREAD

**Issue by ascheider** [BlackbitDigitalCommerce/pimcore-data-director#170]
I have a class field country_layout which is simple dropdown

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/ed30b305-c7d8-4519-89c9-a494cb001717)
![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/9596806a-df23-4603-9a23-86e3d10fceaf)

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/6016485/7ad65d4b-8de5-4983-968e-09f612f28e02)

After the import i get the exception. See Trace

[trace.txt](https://github.com/user-attachments/files/15901223/trace.txt)

The $siblings is loaded as Listing ant then you are trying to add the $item in line 4005 like in array.

**Comment by BlackbitDevs** [comment]
Which DD version?

**Comment by BlackbitDevs** [comment]
Think I found it. Can you please check `3.6.x-dev`?
Have added this:
```php
$siblings = $parentElement->getChildren([AbstractObject::OBJECT_TYPE_OBJECT, AbstractObject::OBJECT_TYPE_VARIANT], true);
            if($siblings instanceof AbstractListing) {
                $siblings = $siblings->load();
            }
```

**Comment by ascheider** [comment]
@BlackbitDevs Yes. Thanks. It helps. I think you can merge it.

**Comment by BlackbitDevs** [comment]
Released 3.6.41.

I have to admit that the "Optimize inheritance" feature is used only by a few clients. If you see any problems with it, please notify me.
